javascript/javascript1/week4/tic_tac_toe/ttt.js

```diff
@@ -0,0 +1,64 @@
+let position = [
```

Comment:
this is not a position, this is a whole board. `position` would be some x,y coordinate for example.